models/maker/maker_accounting.sql

```diff
@@ -0,0 +1,969 @@
+        , (LOWER('0x124c759D1084E67B19a206ab85c4527Fab26c342'), 'IS Ops Wallet', 'Fixed', 'IS-001')
+        , (LOWER('0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F'), 'Data Insights Wallet', 'Fixed', 'DIN-001')
+        , (LOWER('0x2dC0420A736D1F40893B9481D8968E4D7424bC0B'), 'TechOps', 'Fixed', 'TECH-001')
+        , (LOWER('0x2B6180b413511ce6e3DA967Ec503b2Cc19B78Db6'), 'Oracle Gas Cost Multisig + Emergency Fund 1', 'Variable', 'GAS')
+        , (LOWER('0x1A5B692029b157df517b7d21a32c8490b8692b0f'), 'Oracle Gas Cost Multisig + Emergency Fund 2', 'Variable', 'GAS')
+        , (LOWER('0x53CCAA8E3beF14254041500aCC3f1D4edb5B6D24'), 'Oracle Multisig, Emergency Multisig 1', 'Fixed', 'ORA-001')
+        , (LOWER('0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf'), 'Oracle Multisig, Emergency Multisig 2', 'Fixed', 'ORA-001')
+        , (LOWER('0xf737C76D2B358619f7ef696cf3F94548fEcec379'), 'Strategic Finance Multisig', 'Fixed', 'SF-001')
+        , (LOWER('0x3d274fbac29c92d2f624483495c0113b44dbe7d2'), 'Events Multisig', 'Fixed', 'EVENTS-001')
+        , (LOWER('0x34d8d61050ef9d2b48ab00e6dc8a8ca6581c5d63'), 'Foundation Operational Wallet', 'Fixed', 'DAIF-001')
+        , (LOWER('0xbe8e3e3618f7474f8cb1d074a26affef007e98fb'), 'DS Pause Proxy', 'Variable', 'DSPP')
+        , (LOWER('0x73f09254a81e1f835ee442d1b3262c1f1d7a13ff'), 'Interim Multisig', 'Fixed', 'INTERIM')
+        , (LOWER('0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6'), 'SES Multisig (Auditor)', 'Fixed', 'SES-001')
+        , (LOWER('0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad'), 'DUX Auditor Wallet', 'Fixed', 'DUX-001')
+        , (LOWER('0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da'), 'CES Auditor Wallet', 'Fixed', 'CES-001')
+        , (LOWER('0xf482d1031e5b172d42b2daa1b6e5cbf6519596f7'), 'DECO Auditor Wallet', 'Fixed', 'DECO-001')
+        , (LOWER('0xb1f950a51516a697e103aaa69e152d839182f6fe'), 'SAS Auditor Wallet', 'Fixed', 'SAS-001')
+        , (LOWER('0xd1f2eef8576736c1eba36920b957cd2af07280f4'), 'IS Auditor Wallet', 'Fixed', 'IS-001')
+        , (LOWER('0x96d7b01Cc25B141520C717fa369844d34FF116ec'), 'RWF Auditor Wallet', 'Fixed', 'RWF-001')
+        , (LOWER('0x1a3da79ee7db30466ca752de6a75def5e635b2f6'), 'TechOps Auditor Wallet', 'Fixed', 'TECH-001')
+        , (LOWER('0x5F5c328732c9E52DfCb81067b8bA56459b33921f'), 'Foundation Reserves', 'Fixed', 'DAIF-001')
+        , (LOWER('0x478c7ce3e1df09130f8d65a23ad80e05b352af62'), 'Gelato Keepers', 'Variable', 'GELATO')
+        --, (LOWER('0x0048FC4357DB3c0f45AdEA433a07A20769dDB0CF'), 'DSS Blow', 'Variable', 'BLOW')
+    ) AS  t(wallet_address, wallet_label, varfix, code)
+)
+
+-- ********** Helper Tables *********** 
+, treasury_erc20s AS( -- unused CTE #Hosuke commented
+    SELECT '0xc18360217d8f7ab5e7c516566761ea12ce7f9d72' AS contract_address,
+           '0xc18360217d8f7ab5e7c516566761ea12ce7f9d72' AS price_address,
+           18                                           AS decimals,
```

Comment:
@SebVentures Since some CTEs are not used in this file, do you want to remove them? This file is large and be kind of hard to maintain/edit in the future. Do you think this large model can be break into smaller sub-models? (Like divide the final union all parts into different files under this directory)

Reply:
Agree that this query is big and quite dusty (2 years of history). We are still trying to understand how to get it right. No problem to remove unused CTE.

I would also favor breaking it in subparts like maker.accounting being an union of maker.accounting_interest, maker.accounting_opex, maker.accounting_liquididation, .... 

We are starting fresh on ENS, see [this document](https://docs.google.com/document/d/1xS4nXx1G0QCjFS-VdG5yVmVoMa5t1q9_dFZ9N4wGSJ8/edit) so maybe we can work together on this one to see how to leverage Dune the best then later come back to the Maker one?

Reply:
I will try to restructure the sql in this PR, and hopefully come up with a framework we can easily fill in the subparts in the future.

Reply:
This table is used though, isn't it?

Reply:
Sorry, this CTE is used.